pkg/index/job/save/config/config.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2019-2023 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package config
+
+import (
+	"github.com/vdaas/vald/internal/config"
+	"github.com/vdaas/vald/internal/errors"
+)
+
+// GlobalConfig is a type alias of config.GlobalConfig representing application base configurations.
+type GlobalConfig = config.GlobalConfig
+
+// Data represents the application configurations.
+type Data struct {
+	// GlobalConfig represents application base configurations.
+	config.GlobalConfig `json:",inline" yaml:",inline"`
+
+	// Server represent all server configurations
+	Server *config.Servers `json:"server_config" yaml:"server_config"`
+
+	// Observability represents observability configurations.
+	Observability *config.Observability `json:"observability" yaml:"observability"`
+
+	// Save represents auto save indexing service configurations.
+	Save *config.IndexSave `json:"savior" yaml:"savior"`
+}
+
+// NewConfig load configurations from file path.
```

Comment:
```suggestion
// NewConfig loads configurations from the file path.
```

Reply:
sorry... I will apply it!
Thank you for catching it 🙏